3drepo/3drepo.io
# ISSUE #3625 - Custom Tickets Issue Properties

This fixes #3625 

#### Description
Moves everything from the "Properties" accordion into a styled top panel.
Refactored Chip code
Added ChipSelect which is a drop down select that looks like a chip
Added stories for ChipSelect

#### Test cases
- Look at the panel in different tickets. Check the styling doesn't turn messy
- Make sure you check that unset due dates and unassigned look good.
- Ensure the autoscroll when expanding a module works
- Ensure the New Ticket screen has no styling errors

I was following this [mockup](https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=340%3A379&t=JjDgq4QoGIPasvuM-0)
